lib/Graph/Nodes.cpp

```diff
@@ -299,6 +299,17 @@ void ConvolutionGradNode::verify() const {
                     Kernels_, Strides_, Pads_, Group_);
 }
 
+void ConvertToNode::verify() const {
+  assert(getInput().dims() == getResult().dims() && "Shape must be the same");
+  TypeRef srcTy = getInput().getType();
+  (void)srcTy;
+  TypeRef dstTy = getResult().getType();
+  (void)dstTy;
+  assert(srcTy != dstTy && "Nothing to convert");
+  assert(!srcTy->isQuantizedType() && !dstTy->isQuantizedType() &&
+         "Quantized conversion should use Dequantize, Quantize and Rescale");
```

Comment:
Why don't we want to use this for quantization?

Reply:
Fair enough. One thing then is that `ConvertTo` is named somewhat generically, like it could apply to `Quantize`/`Dequantize`. Not a big deal but led me to believe it could be used on Quantized types. Perhaps the docstring can specify that at least?

Also, if we add other quantization bit widths, would we extend `Quantize`/`Dequantize` in the same way this node behaves (switching between types to determine what to do)? Would `Rescale` move between these different quantization widths? Maybe it's out of scope here, but seems related.

Reply:
> Perhaps the docstring can specify that at least?

I'll post a patch in that direction.

> Also, if we add other quantization bit widths, would we extend Quantize/Dequantize in the same way this node behaves (switching between types to determine what to do)? 

I assume your talking about the implementation in the backends, if that's the case, I would expect so. Now, I am not a huge fan of that approach and I suggested ways to alleviate it in the compiler (optimization and interpreter, however, codegen would still need to go through that pain) see https://github.com/pytorch/glow/issues/1329#issuecomment-425284818, but that's the status we are now and given we don't add a type very often we might continue like this for quite some time :).